state/upgrades.go

```diff
@@ -1272,6 +1273,8 @@ func MigrateLeasesToGlobalTime(pool *StatePool) error {
 	return runForAllModelStates(pool, migrateModelLeasesToGlobalTime)
 }
 
+const initialLeaderClaimTime = time.Minute
```

Comment:
This appears duplicated. Can we use a single sourced value?

Reply:
Actually, it needed to be public in state - the other way produces an import cycle.